src/tm-threads.c

```diff
@@ -2190,14 +2190,26 @@ void TmThreadsSetThreadTimestamp(const int id, const SCTime_t ts)
     int idx = id - 1;
     Thread *t = &thread_store.threads[idx];
     t->pktts = ts;
-    struct timeval systs;
-    gettimeofday(&systs, NULL);
-    t->sys_sec_stamp = (uint32_t)systs.tv_sec;
+    struct timeval tv;
```

Comment:
I see that in same situations we're still using `timeval`. What's the criteria, now? Is `timeval` only for local variables, and for setting or initializing an `SCTime_t`?

Reply:
we still need it for gettimeofday